src/child.rs

```diff
@@ -195,5 +205,7 @@ mod ffi {
     extern {
         pub fn pivot_root(new_root: *const c_char, put_old: *const c_char)
             -> c_int;
+
+        pub fn setns(fd: c_int, nstype: c_int) -> c_int;
```

Comment:
`nix::sched::setns` appeared in `nix==0.5.0` and it breaks several places in `unshare`. So I think updating `nix` should be another PR.